src/hospital/covid_examinations/models/patient.py

```diff
@@ -0,0 +1,20 @@
+"""Patient model."""
+from django.db import models
+
+from person import Person
+from department import Department
+
+
+class Patient(models.Model):
+    department = models.ForeignKey(Department,
+                                   null=False,
+                                   blank=False,
+                                   related_name="patients",
+                                   on_delete=models.CASCADE)
+
+    personal_information = models.ForeignKey(Person,
+                                             null=False,
+                                             blank=False,
+                                             on_delete=models.CASCADE,
+                                             related_name=
+                                             "treating_departments")
```

Comment:
I don't like this related name
I would go with `partients` (same as you used in department related name)